pageserver/src/tenant/gc_block.rs

```diff
@@ -1,10 +1,16 @@
-use std::collections::HashMap;
+use std::{collections::HashMap, time::Duration};
 
+use super::{remote_timeline_client::index::GcBlockingReason, tasks};
+use tokio_util::sync::CancellationToken;
 use utils::id::TimelineId;
 
-use super::remote_timeline_client::index::GcBlockingReason;
+type TimelinesBlocked = HashMap<TimelineId, enumset::EnumSet<GcBlockingReason>>;
 
-type Storage = HashMap<TimelineId, enumset::EnumSet<GcBlockingReason>>;
+#[derive(Default)]
+struct Storage {
+    pub timelines_blocked: TimelinesBlocked,
+    pub tenant_blocked: bool,
```

Comment:
These should not be accessed from anywhere else than this module, so let's drop the `pub`:

```suggestion
    timelines_blocked: TimelinesBlocked,
    tenant_blocked: bool,
```

Reply:
526d8a094115afe5531622a43e02b734c3b1fc40